From 2485ded64158b4949815f81753aa53594538bb75 Mon Sep 17 00:00:00 2001
From: Gerrit Pape <pape@smarden.org>
Date: Thu, 3 May 2007 09:21:13 +0000
Subject: [PATCH 06/13] Don't define WANT_SSP.

It causes similar problems as WANT_STACKGAP back in Dec 2004, see
http://bug.debian.org/374349
---
 dietfeatures.h |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

Index: dietlibc-0.33~cvs20110918/dietfeatures.h
===================================================================
--- dietlibc-0.33~cvs20110918.orig/dietfeatures.h	2011-11-03 21:33:57.000000000 +0100
+++ dietlibc-0.33~cvs20110918/dietfeatures.h	2011-11-03 21:51:54.000000000 +0100
@@ -135,7 +135,7 @@
  * compiled with -fstack-protector against it, the binary will segfault
  * when calling that code. */
 #if (__GNUC__>4) || ((__GNUC__==4) && (__GNUC_MINOR__>=1))
-#define WANT_SSP
+/* #define WANT_SSP */
 #endif
 
 
